rpm/ngfd-tone-generator.conf

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <!-- Allow any connection to receive the message -->
+  <policy context="default">
```

Comment:
Do we really need to allow to own this name to everybody? Isn't it enough to only allow it for the daemon? (root user or something)